### ISSUE TITLE
Particles in purgatory

### ISSUE THREAD

**Issue by rmcdermo** [firemodels/fds#7146]
Jason, I'm trying to develop a case where we mimic the flow of PIV particles.  I have a hunch this is affecting one of the validation cases I'm working on, the Juelich thermal plume, which is a small 75 W hot plate in a mostly closed compartment.  I've simplified the case below.

In this case, the particles are very small and eventually the LP%MASS becomes less than TWO_EPSILON_EB.  But they are not removed from the domain.  So, it seems this is a situation in which the evaporating particles are too small to move, but they are in a cool flow and thus are not evaporating away.  I think if the particles are too small to move, they should be killed.

[test.fds.txt](https://github.com/firemodels/fds/files/2832779/test.fds.txt)

In my actual problem, I'm going to work on increasing the particle PWT, but wanted to post this issue while I had it cornered.  Thanks.

**Comment by drjfloyd** [comment]
MINIMUM_DIAMETER sets the minimum of the particle distribution.  KILL_RADIUS is set to 0.5 % of the mass of the minimum diameter. In this case that mass is ~E-18.  We did this approach since before we were killing small particles in fine sprays the moment they dropped below the minimum diameter.  Try upping MINIMUM_DIAMETER to 0.98.

**Comment by rmcdermo** [comment]
I can get this case to work by fiddling with the numbers.  But I think this state needs to be avoided automatically.  If the particles no longer move they should be removed from the domain.  When LP%MASS is below TWO_EPSILON_EB I think it is ok to lose that energy.

**Comment by drjfloyd** [comment]
Why are the particles not moving? In the gas phase a really tiny particle should just move with the gas.  

TWO_EPSILON_EB is 4E-16 kg which is a 1 micron droplet.  That is too large to kill if we are dealing with some high pressure water mist or fog type systems.

**Comment by rmcdermo** [comment]
Because of this line at about line 1369 part.f90

```
      ! Throw out particles that have run out of mass.

      IF (.NOT.LPC%MASSLESS_TRACER .AND. (R_D<=0._EB .OR. (.NOT.LPC%STATIC .AND. LP%MASS<=TWO_EPSILON_EB))) CYCLE PARTICLE_LOOP
```

**Comment by drjfloyd** [comment]
Ah. Well that line is probably not a good line to have.

**Comment by drjfloyd** [comment]
I think that line should just be deleted and we kill based on the kill radius.

**Comment by drjfloyd** [comment]
I think we want to combine line 1369 and 1406 into something like below at 1369 and delete 1406:

IF (.NOT. LPC%MASS_TRACER .AND. (R_D < LPC%KILL_RADIUS .OR. (LPC%STATIC .AND. .NOT.LP%EMBER) .OR LP%PWT < TWO_EPSILON_EB) CYCLE PARTICLE_LOOP

**Comment by rmcdermo** [comment]
Looks good to me.  I'll give it a try and report back.

**Comment by rmcdermo** [comment]
Should be fixed with PR #7149
